doc/_static/mpl.css

```diff
@@ -986,7 +986,7 @@ div.sphx-glr-download a {
    background-image: none !important;
 }
 
-p.sphx-glr-signature a.reference.external {
```

Comment:
did you mean to drop the css for `a.reference.external`?

Reply:
Yes, because I want to hide not only the `<a>` tag but the outer `<p>` tag.  
It can of course be discussed whether the search keywords should still be shown on the page. In my opinion that just distracts and is not really useful.